components/hab/src/main.rs

```diff
@@ -451,6 +452,13 @@ fn sub_origin_secret_list(ui: &mut UI, m: &ArgMatches<'_>) -> Result<()> {
     command::origin::secret::list::start(ui, &url, &token, &origin)
 }
 
+fn sub_origin_create(ui: &mut UI, m: &ArgMatches<'_>) -> Result<()> {
+    let origin = origin_param_or_env(&m)?;
```

Comment:
`<ORIGIN>` is a required argument [per the clap definition](https://github.com/habitat-sh/habitat/pull/6758/files#diff-ee10e75b553edc813cf07f6a5036374cR248), so falling back to the env var seems contradictory. Our typical pattern here has been something like:
```suggestion
    let origin = m.value_of("ORIGIN").expect("required ORIGIN");
```

Reply:
I'm guessing this is copy-paste from some other command

Reply:
Corrected.